docker/rsem/Dockerfile

```diff
@@ -1,18 +1,51 @@
-FROM centos:centos7
+FROM ubuntu:17.04
+MAINTAINER Jishu Xu <jishuxu@broadinstitute.org>
+LABEL software="rsem"
+LABEL version="1.3.0"
+LABEL description="RNA-seq gene expression quantification tools"
+LABEL website="https://deweylab.github.io/RSEM/"
 
-LABEL maintainer="Jishu Xu <jishuxu@broadinstitute.org>" \
-  software="RSEM" \
-  version="v1.3.0" \
-  Description="This image is used for running RSEM RNA-Seq analysis tools " \
-  website="https://github.com/deweylab/RSEM"
+WORKDIR /usr/local/
+
+RUN apt-get update
+# Install compiler, perl , R and stuff
+RUN apt-get update --fix-missing \
+    && apt-get install -y \
+        build-essential \
+        gcc-multilib \
+        apt-utils \
+        zlib1g-dev \
+        libxml2-dev \
+        curl \
+        wget \
+		git \
+		perl \
+		perl-base \
+        libbz2-dev \
+        cmake automake \
+        libboost-all-dev \
+        libncurses5-dev \
+        r-base \
+        r-base-core \
+        r-base-dev \
+        libncurses5-dev \
+		bowtie \
+		bowtie2 
 
+# Compile and install STAR
+WORKDIR /usr/local
+RUN wget https://github.com/alexdobin/STAR/archive/2.5.3a.tar.gz && \
+    tar -xf 2.5.3a.tar.gz
+WORKDIR STAR-2.5.3a/bin/Linux_x86_64_static
+RUN cp STAR /usr/local/bin
+
+# Install RSEM 
 WORKDIR /usr/local/
-RUN yum groupinstall -y 'Development Tools' 
-RUN yum install -y perl-Env 
-RUN yum install -y zlib-devel
+RUN pwd
```

Comment:
Is this needed, or leftover from debugging?

Reply:
debugging, removed